a-tier-bonding/src/components/header/header.js

```diff
@@ -49,7 +50,7 @@ function Header() {
         <img src={earthLogo} style={{ height: "75px", width: "75px" }} alt="Logo"/>
         <ButtonGroup color='inherit' size="large" variant="text">
           <Button component={Link} to='/'  style={{ fontSize: '2vh', fontFamily: "Nevis" }}>  Home  </Button>
-          <Button component={Link} to='/schedule' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Schedule  </Button> 
+          {/* <Button component={Link} to='/schedule' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Schedule  </Button>  */}
           <Button component={Link} to='/team' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Team </Button> 
           <Button href='https://www.twitch.tv/phieki' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Twitch </Button> 
           <Button href='https://tiltify.com/@phieki/letsagame-earth/donate' style={{ fontSize: '2vh', fontFamily: "Nevis" }}> Donate  </Button> 
```

Comment:
I just created a new campaign, can we change this link to https://donate.tiltify.com/@phieki/letsagamelegacy ?

Reply:
(For both here and the above component)